store/service/service.go

```diff
@@ -28,6 +28,17 @@ type serviceStore struct {
 	Client pb.StoreService
 }
 
+func (s *serviceStore) Init(opts ...store.Option) error {
+	for _, o := range opts {
+		o(&s.options)
+	}
+	s.Namespace = s.options.Namespace
```

Comment:
Something irks me about the repetition/duplication of `serviceStore` struct fields and `store.Option` fields 🤷‍♂️

Reply:
I go back and forth between in. Being able to explictly see the things you use is useful but agree that there's duplication which is actually error prone. Not just limited to this implementation. Probably one for a wider cleanup.